### PR TITLE
Submission bug

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #379:*
https://github.com/bcgov/met-public/issues/#379

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by tiagograf-AOT** [comment]
It doesn't look like this would solve the problem, were you able to reproduce the issue?

File: met-api/src/met_api/services/comment_service.py [hunk 1]
```diff
@@ -64,20 +64,19 @@ def extract_comments(cls, survey_submission: SubmissionSchema, survey: SurveySch
...
         if len(components) == 0:
             return []
-
         # get the 'key' for each component that has 'inputType' text and filter out the rest.
         comments_keys = [
             component.get('key', None) for component in components
             if component.get('inputType', None) == 'text']
-
         submission = survey_submission.get('submission_json', {})
         comments_texts = [submission.get(key, None) for key in comments_keys]
```
Comment (tiagograf-AOT) [inline met-api/src/met_api/services/comment_service.py:72]: I'm gonna throw a different approach and leave it to Jad to decide, but when assigning comments_texts you can filter out the blank ones something like:  
        comments_texts = [submission.get(key, None) for key in comments_keys
        if submission.get(key, None) != '']

**Review by tiagograf-AOT — APPROVED** [review]

File: met-api/src/met_api/services/comment_service.py (hunk 1 above)
Comment (jadmsaadaot) [reply met-api/src/met_api/services/comment_service.py:72]: yup this is definitely cleaner and more performant

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Please filter empty texts Tiago's way

File: met-api/src/met_api/services/comment_service.py (hunk 1 above)
Comment (djnunez-aot) [reply met-api/src/met_api/services/comment_service.py]: good idea

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**